File: backend/ReptiRealm/src/ReptiRealm/Models/Shed.cs
using System.ComponentModel.DataAnnotations;

namespace ReptiRealm.Models
{
    public class Shed : BaseEntity
    {
        [Required]
        public required DateOnly Date {  get; set; }
        [Required]
        public required char BlueOrShed { get; set; }
        [Required]
        public required char Rating { get; set; }
        public string? Comment { get; set; }
    }
}

File: App/lib/models/shed.dart
class Shed {
  final String? id;
  final DateTime date;
  final String blueOrShed;
  final String rating;
  final String? comment;

  Shed({
    this.id,
    required this.date,
    required this.blueOrShed,
    required this.rating,
    this.comment,
  });
}
